source/includes/fundamentals/code-snippets/indexes.rs

```diff
@@ -67,6 +69,68 @@ async fn main() -> mongodb::error::Result<()> {
     println!("Created index:\n{}", idx.index_name);
     // end-text
 
+    let my_coll: Collection<Document> = client.database("sample_training").collection("posts");
+
+    // begin-atlas-model
+    let def = doc! { "mappings": doc! {
+        "dynamic": false,
+        "fields": {
+            "body": {"type": "string"},
+            "date": {"type": "date"} 
+        }
+    }};
+
+    let idx_model = SearchIndexModel::builder()
+        .definition(def)
+        .name("example_index".to_string())
+        .build();
+    // end-atlas-model
+
+    // begin-atlas-create-one
+    let idx_model = SearchIndexModel::builder()
+        .definition(doc! { "mappings": doc! {"dynamic": true} })
```

Comment:
S:(nit) noticed that formatting isnt totally consistent throughout the file, could fix by running the formatter
```suggestion
        .definition(doc! { "mappings": doc! { "dynamic": true } })
```